lib/spaceship/client.rb

```diff
@@ -247,10 +247,12 @@ def send_request(method, url_or_path, params, headers, &block)
     end
 
     def parse_response(response, expected_key = nil)
-      if expected_key
-        content = response.body[expected_key]
+      content = if response.body
```

Comment:
In general we try to avoid the use of the return value of the `if` statement. Could you rewrite it so it's easier for non-Ruby developers to understand?